### PR TITLE
Make Mint page simpler

### PR DESCRIPTION
**Pull request by tinaroh**
- Removed total minted (maybe we can bring it back in some stats page?)
- No referral bonus button if there's nothing pending
- Moved the invite image to invite flow and adjusted some font sizes

screenshot_20180930-203944_raha test

screenshot_20180930-212105_raha test

### CODE REVIEW

File: src/components/shared/MintButton.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
       }
     : undefined;
   return (
-    <React.Fragment>
+    <View>
```
Comment (rahulgi) [inline src/components/shared/MintButton.tsx]: What's the difference?

File: src/components/pages/AccountSettings/SignOut.tsx [hunk 2]
```diff
@@ -17,16 +18,29 @@ type Props = DispatchProps & OwnProps;
...
+      <Text style={styles.body}>
+        You will have to receive an SMS one-time-password to sign in again.
       </Text>
-      <Button style={styles.row} title="Sign Out" onPress={signOut} />
+      <Button title="Sign Out" onPress={signOut} />
     </ScrollView>
   );
 };
 
+export const styles = StyleSheet.create({
```
Comment (rahulgi) [inline src/components/pages/AccountSettings/SignOut.tsx]: Why is this being exported?

File: src/components/pages/Wallet.tsx [hunk 3]
```diff
@@ -178,25 +208,15 @@ const financesSectionStyle: ViewStyle = {
 
 const donationSectionStyle: ViewStyle = {
   flexDirection: "row",
-  alignItems: "center",
-  justifyContent: "flex-start"
+  alignItems: "center"
 };
 const balanceSectionStyle: ViewStyle = {};
-const moneyElementStyle: ViewStyle = { marginRight: 20 };
+const moneyElementStyle: ViewStyle = {};
```
Comment (rahulgi) [inline src/components/pages/Wallet.tsx]: should we just remove this?

File: src/components/shared/MintButton.tsx (hunk 1 above)
Comment (tinaroh) [reply src/components/shared/MintButton.tsx]: Grouping, if it's React.Fragment then spaced-evenly applies

File: src/components/pages/AccountSettings/SignOut.tsx (hunk 2 above)
Comment (tinaroh) [reply src/components/pages/AccountSettings/SignOut.tsx]: oh i was copypastaing

**Pull request merged**